pkg/autoscaler/statserver/server_test.go

```diff
@@ -166,6 +174,49 @@ func TestServerDoesNotLeakGoroutines(t *testing.T) {
 	server.Shutdown(time.Second)
 }
 
+func BenchmarkStatServer(b *testing.B) {
+	encodingName := map[bool]string{
+		false: "gob",
+		true:  "json",
+	}
+
+	for _, jsonEncoding := range []bool{false, true} {
```

Comment:
I'd make the map above `map[string]bool` and then iterate over the map and make this `for encoding, jsonEncoding := range encodingName`

Reply:
ugh, I spent like 10 minutes trying and failing to figure out exactly that nice way of doing this :D.

Reply:
given it's a benchmark we can just
```
for _, enc := range []{"json", "gob"} {
jc := enc == "json"
```
and avoid the map, which is kind of ugly now :)

Reply:
oh I missed this before, yeah that's nice too - oh well